### ISSUE TITLE
⌨ Signout button incorrect focus styles

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#11981]
## ⌨ Accessibility Issue

The signout button has no specific focus styles (blue background, black text) making it differ from all other interactive elements on the site.

Also you can't get to it with keyboard arrow navigation. Need to tab to it.

## 🦋 Expected Behaviour

Signout button styling is consistent with other interactive elements.

## 📋 Steps to Reproduce

1. Login as any user role
2. Using only keyboard, tab to the signout button in the account menu dropdown
3. Observe the default focus styles

## 📸 Screenshot

![2024-11-15_08-17](https://github.com/user-attachments/assets/83718e82-cc7f-47dc-bb89-6328cfa80ed3)

## 🙋‍♀️ Proposed Solution

Use the same styles we use for other text links/buttons.

## ✅ Acceptance Criteria

- [ ] Signout button styles are consistent with other interactive elements
- [ ] Keyboard arrow key navigation within the sub-menu can get you to the sign out button

**Comment by tristan-orourke** [comment]
Likely also related to [#11991](https://github.com/GCTC-NTGC/gc-digital-talent/issues/11991)
